assets/src/edit-story/components/form/dateTime/timePicker.js

```diff
@@ -0,0 +1,367 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable header/header */
+/* Disable reason: This file is based on Gutenberg Datepicker */
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import moment from 'moment';
+import styled, { css } from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
+
+const DateTimeWrapper = styled.div`
+  margin-bottom: 1em;
+`;
+
+const Fieldset = styled.fieldset`
+  margin-top: 0.5em;
+  position: relative;
+`;
+
+const Legend = styled.legend`
+  position: absolute;
+  top: -999em;
+  left: -999em;
+`;
+
+const InputRow = styled.div`
+  display: flex;
+`;
+
+const inputCSS = css`
+  min-height: 30px;
+  box-shadow: 0 0 0 transparent;
+  transition: box-shadow 0.1s linear;
+  border-radius: 4px;
+  border: 1px solid #7e8993;
+  box-sizing: border-box;
+  height: 28px;
+  vertical-align: middle;
+  background-color: #fff;
+  color: #32373c;
+`;
+
+const NumberInput = styled.input`
+  ${inputCSS}
+  padding: 2px;
+  margin-right: 4px;
+  text-align: center;
+  width: ${({ width }) => width}px;
+`;
+
+const SelectInput = styled.select`
+  font-size: 13px;
+  line-height: 1.38461538;
+  color: #32373c;
+  border-color: #7e8993;
+  box-shadow: none;
+  border-radius: 3px;
+  padding: 3px 24px 3px 8px;
+  min-height: 28px;
+  max-width: 25rem;
+  vertical-align: middle;
+  appearance: none;
+  background: #fff
+    url(data:image/svg+xml;charset=US-ASCII,%3Csvg%20width%3D%2220%22%20height%3D%2220%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cpath%20d%3D%22M5%206l5%205%205-5%202%201-7%207-7-7%202-1z%22%20fill%3D%22%23555%22%2F%3E%3C%2Fsvg%3E)
```

Comment:
Is this the best way to bring in a SVG? Should be a file in our codebase?

Reply:
We'll have to replace the GB date picker anyway, no need to modify copied files